src/Elastic.CommonSchema.NLog/EcsLayout.cs

```diff
@@ -534,7 +534,7 @@ private Event GetEvent(LogEventInfo logEventInfo)
 				Action = !string.IsNullOrEmpty(eventAction) ? eventAction : null,
 				Id = EventId?.Render(logEventInfo),
 				Code = eventCode,
-				Kind = !string.IsNullOrEmpty(eventKind) ? eventKind : null,
+				Kind = !string.IsNullOrEmpty(eventKind) ? eventKind : "event",
```

Comment:
Please put this logic into the `EventKind`-Layout in your own solution. What is good default for you, might be noise for others.

Ex. one can do this in NLog `eventKind="${event-property:EventKind:whenEmpty=event}"`

Reply:
My understanding is that on reading the docs, event kind should be set in the logs. However if the noise is a genuine concern, perhaps we include multiple layouts in the package 1 for each kind.